### PR TITLE
observatory_lib

### PR DESCRIPTION
**Pull request by devoncarew**
- normalize package name from `observatory_gen` and `observatory_api_dart` to `observatory_lib`
- format the generated dart library

@danrubel

### CODE REVIEW

**Comment by sethladd** [comment]
Is this really observatory or the base service protocol?

**Comment by devoncarew** [comment]
I generally refer to it as the observatory protocol, but if the vm team prefers 'vm service protocol', sgtm. The dart lib would then be named something like `vm_service_protocol_lib`? `vm_service_protocol`? I think leaving `vm` out of the name would make the name too generic.

**Comment by devoncarew** [comment]
`vm_service_lib`?

File: dart/lib/observatory_lib.dart [hunk 1]
```diff
@@ -1,18 +1,20 @@
 // This is a generated file.
 
-library observatory_gen;
+/// A library to access the Observatory API.
```
Comment (danrubel) [inline dart/lib/observatory_lib.dart]: Observatory -> VM Service

**Comment by danrubel** [comment]
Yes... vm_service_lib ... that works for me. And good idea to auto format generated code. LGTM. Thanks!

**Comment by devoncarew** [comment]
Updated to `vm_service_lib`.

**Comment by danrubel** [comment]
LGTM

**Pull request merged**